src/constants/referralsConstants.js

```diff
@@ -33,3 +33,4 @@ export const FETCHING_REFERRAL_REWARD_AMOUNT = 'FETCHING_REFERRAL_REWARD_AMOUNT'
 export const SET_REFERRAL_REWARD_ISSUER_ADDRESSES = 'SET_REFERRAL_REWARD_ISSUER_ADDRESSES';
 export const SET_PILLAR_REWARD_CAMPAIGN_STATUS = 'SET_PILLAR_REWARD_CAMPAIGN_STATUS';
 export const SET_1WORLD_CAMPAIGN_STATUS = 'SET_1WORLD_CAMPAIGN_STATUS';
+export const OTP_DIGITS = 4;
```

Comment:
I'm assuming this will always be 4...

Reply:
yeah... that's why it's a constant :D previously it was hardcoded in several places in the code